docs/editor/vscode-web.md

```diff
@@ -89,6 +89,14 @@ In some cases, you will want to access a different environment that has the abil
 
 The GitHub Repositories extension makes it easy for you to clone the repository locally with the **GitHub Repositories: Continue Working on...** command available from the Command Palette (`kbstyle(F1)`) or by clicking on the Remote indicator in the Status bar.
 
+## Bring your own compute with the VS Code Server
```

Comment:
How about "Use your own compute power with the VS Code Server"

Reply:
I like that! Updated.